pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -10,14 +10,14 @@
 # hardware problems with a new one.
 
 # Configuration
-{ stdenv, version
+{ lib, stdenv, version
```

Comment:
@jonringer This was a change I had to do manually to fix the evaluation error at some point.